### ISSUE TITLE
Crashes

### ISSUE THREAD

**Issue by FXUDarkNova** [xindawndev/xindawn-android-airplay-mirroring-sdk#1]
Hi there,
Everytime I run this and hit the start button the application crashes why is this?

**Comment by KushtrimPacaj** [comment]
Is your device armeabi-v7a?

**Comment by FXUDarkNova** [comment]
Its Amlogic ARM® Cortex®-A53(ARMv8) trying on a odroid c2

**Comment by KushtrimPacaj** [comment]
Try running it in a device with armeabi-v7a.

Check the libs folder, the shared library is only for that ABI.

**Comment by FXUDarkNova** [comment]
Oh ok do you not have anything available for Amlogic ARM® Cortex®-A53(ARMv8)?

**Comment by KushtrimPacaj** [comment]
Nope. But I'm not the developer of the library. Wait, maybe he/she will respond

**Comment by FXUDarkNova** [comment]
Ok well thank you for your help

**Comment by FXUDarkNova** [comment]
Got it working do you know if i purchase mac license from xindawn site that comes with everything need for android?

**Comment by xindawndev** [comment]
Only Android active code can be used for Android platform

2017-01-28 0:19 GMT+08:00 FXUDarkNova <notifications@github.com>:

**Comment by sunflower8086** [comment]
Hi there,
Everytime I run this and click the start button the application crashes why is this?And I find when call the function in jni will never got a response,could anyone can help to provide a solution,thanks a lot!
ret = startMediaRender(friendname,objpath,1280,720,47000,7000,128*1024,RenderApplication.getInstance());

**cpu info as below:**
processor	: 3
model name	: ARMv7 Processor rev 5 (v7l)
BogoMIPS	: 38.40
Features	: swp half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 evtstrm 
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xc07
CPU revision	: 5

Hardware	: Qualcomm Technologies, Inc MSM8909
Revision	: 0000
Serial		: 0000000000000000
Processor	: ARMv7 Processor rev 5 (v7l)

**Comment by sunflower8086** [comment]
the crash info as below:

pid: 13012, tid: 13260, name: Thread-3  >>> com.xindawn <<<
signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
    r0 00000000  r1 000033cc  r2 00000006  r3 00000008
    r4 8f8a9978  r5 00000006  r6 8f8a9920  r7 0000010c
    r8 00000000  r9 8f313ef9  sl 8f40a2a7  fp 8f40a2b8
    ip 00000054  sp 8f886940  lr ab418887  pc ab41b108  cpsr 200f0010

backtrace:
    #00 pc 0004a108  /system/lib/libc.so (tgkill+12)
    #01 pc 00047883  /system/lib/libc.so (pthread_kill+34)
    #02 pc 0001db59  /system/lib/libc.so (raise+10)
    #03 pc 00019651  /system/lib/libc.so (__libc_android_abort+34)
    #04 pc 000172c8  /system/lib/libc.so (abort+4)
    #05 pc 00342679  /data/app/com.xindawn-1/lib/arm/libmediaserver.so (_ZN9__gnu_cxx27__verbose_terminate_handlerEv+220)
    #06 pc 0031782b  /data/app/com.xindawn-1/lib/arm/libmediaserver.so (_ZN10__cxxabiv111__terminateEPFvvE+2)
    #07 pc 0031785b  /data/app/com.xindawn-1/lib/arm/libmediaserver.so (_ZSt9terminatev+10)
    #08 pc 00317525  /data/app/com.xindawn-1/lib/arm/libmediaserver.so (__cxa_throw+132)
    #09 pc 002f51f5  /data/app/com.xindawn-1/lib/arm/libmediaserver.so
    #10 pc 002f7573  /data/app/com.xindawn-1/lib/arm/libmediaserver.so
    #11 pc 0002e677  /data/app/com.xindawn-1/lib/arm/libmediaserver.so
    #12 pc 0002ca5f  /data/app/com.xindawn-1/lib/arm/libmediaserver.so
    #13 pc 00032785  /data/app/com.xindawn-1/lib/arm/libmediaserver.so (XinDawn_StartMediaServer+516)
    #14 pc 000064b1  /data/app/com.xindawn-1/lib/arm/libjniinterface.so
    #15 pc 000fe1f5  /data/app/com.xindawn-1/oat/arm/base.odex (offset 0xf7000)
